notebook/notebook/templates/index.html

```diff
@@ -10,9 +10,17 @@
       <h1>Hail Notebooks</h1>
       <form action="{{ form_action_url }}", method="post">
         <p>
-          <label for="password">Class Key:</label>
+          <label>Class Key:</label>
```

Comment:
I think the `for` link should stay? It still describes the input box named password.

Reply:
I got unhappy highlighting in IntelliJ. I put it back